gradle.properties

```diff
@@ -10,3 +10,4 @@ ossrhUsername=
 ossrhPassword=
 archivaUsername=
 archivaPassword=
+snappydata.enterprise=true
```

Comment:
@hemanthmeka this is not correct. We should always keep the project gradle.properties as OSS. Add in your ~/.gradle/gradle.properties if required.

Reply:
didn't notice this being pushed while merging. removed this line in new commit. Thanks